script/src/verify.rs

```diff
@@ -283,6 +304,62 @@ impl<'a, DL: DataLoader> TransactionScriptsVerifier<'a, DL> {
         Ok(cycles)
     }
 
+    fn verify_type_id(
+        &self,
+        script_group: &ScriptGroup,
+        max_cycles: Cycle,
+    ) -> Result<Cycle, ScriptError> {
+        if max_cycles < TYPE_ID_CYCLES {
+            return Err(ScriptError::ExceededMaximumCycles);
+        }
+        // TYPE_ID script should only accept one argument,
+        // which is the hash of all inputs when creating
+        // the cell.
+        if script_group.script.args.len() != 1 || script_group.script.args[0].len() != 32 {
+            return Err(ScriptError::ValidationFailure(-1));
```

Comment:
1. Can we put `verify_type_id` to an independent module, like `TypeIdVerifier`, so that makes it looks more like an independent script?
2. Declare the exit-codes as constant variables.

Reply:
Done